### PR TITLE
Queue API

### PR DESCRIPTION
**Pull request by arranubels**
I added the Queue API to this.

### CODE REVIEW

**Comment by yosida95** [comment]
gofmt plz

File: jenkins.go [hunk 1]
```diff
@@ -133,3 +133,10 @@ func (jenkins *Jenkins) GetBuildConsoleOutput(build Build) ([]byte, error) {
 	defer res.Body.Close()
 	return ioutil.ReadAll(res.Body)
 }
+
+// GetJob returns a job which has specified name.
+func (jenkins *Jenkins) GetQueue() (queue Queue, err error) {
+	err = jenkins.get(fmt.Sprintf("/queue/api/json"), nil, &queue)
```
Comment (yosida95) [inline jenkins.go]: /api/json prefix is automatically appended in buildUrl method.  So appending it is unnecessary here.

**Comment by yosida95** [comment]
.gitignore added in this PR is not mine. Remove it please.

**Comment by arranubels** [comment]
HI Yosida, I have removed it from the pull-request-branch. If I understand this system correctly that should have removed it from the pull request. Sorry about that. Please verify.

**Comment by yosida95** [comment]
I confirmed that .gitignore is removed.  Thanks.

I'll merge this Pull Request when below conditions described in my
before comments are met:
- format source code using gofmt
- remove api/json suffix from URL in (*Jenkins).GetQueue()
- do git rebase to meld your changes in one commit

I'm sorry about my many demands to you.


-- 
==============================
Name: Kohei Yoshida
Mail:   kohei@yosida95.com
Site:   http://yosida95.com/
==============================

**Comment by arranubels** [comment]
I believe the URL suffix is required. Please see new pull request.

**Pull request closed without merging**